pkg/city/city.go

```diff
@@ -29,3 +37,48 @@ func (c *City) GetStopsByID() map[uint64]*GraphNode {
 func (c *City) GetTimeBounds() TimeBounds {
 	return c.cityData.GetTimeBounds()
 }
+
+func (c *City) GetLinesForStop(stopID uint64, chipPerRowSize int) []string {
+	lines, ok := c.linesByStopID[stopID]
+
+	if !ok {
+		return []string{}
+	}
```

Comment:
These probably shouldn't be separated, because the context of the first line is required for the if statement to make sense.
```suggestion
	lines, ok := c.linesByStopID[stopID]
	if !ok {
		return []string{}
	}
```